### ISSUE TITLE
failing tests / invalid (X)HTML

### ISSUE THREAD

**Issue by t4k** [Coral-erm/Coral#279]
While trying to work with the automated tests, I have had some trouble with them. It seems to come down to some invalid XHTML.

Here is one example, in the Licensing menu in the header:
```
    <a href='ajax_forms.php?action=getLicenseForm&height=265&width=260&modal=true&newLicenseID=' class='thickbox’ id='newLicense’>
        <div class="main-menu-link">
            <img src="images/menu/icon-plus-square.png" />
            <span>New License</span>
        </div>
    </a>
```

We have a `<div>` within an `<a>` which is not valid XHTML. This causes the tests to fail when I’m using Chrome and ChromeDriver. The error is:
> Element \<a href="ajax_forms.php?action=getLicenseForm&amp;height=265&amp;width=260&amp;modal=true&amp;newLicenseID=" class="thickbox" id="newLicense">…\</a> is not clickable at point (89, 99). Other element would receive the click: \<td style="width:870px;height:19px;" id="main-menu-titles" colspan="2">…\</td>

This led me down the path of trying out the whole page in the W3C Validator service https://validator.w3.org/ and the results were troubling. We’ve got a lot of issues. I only tried the main Licenses page, but I imagine this is rampant.

**Comment by veggiematts** [comment]
Also, maybe we shouldn't be using XHTML as a doctype in the first place.

For example, we're using the nav element, which is not an XHTML element, but an HTML5 one.

`<nav class="language-select">Change language:`

About your example, it would be ok in HTML5:

> HTML 5 states that the a element "may be wrapped around entire paragraphs, lists, tables, and so forth, even entire sections, so long as there is no interactive content within (e.g. buttons or other links)".

https://www.w3.org/TR/html5/text-level-semantics.html#the-a-element

But whatever the doctype is, I agree we should produce valid content according to it.

**Comment by t4k** [comment]
Yeah, I saw that it would be valid HTML5 when I was looking into it.

Hopefully if we change the doctype it might make the automated tests pass for that HTML structure.

I hope we don’t have other HTML that would be invalid HTML5, though.
